pkg/ctrdtaskapi/update.go

```diff
@@ -15,3 +16,10 @@ type PolicyFragment struct {
 	// fragment and any additional information required for validation.
 	Fragment string `json:"fragment,omitempty"`
 }
+
+type ContainerMount struct {
+	HostPath      string
+	ContainerPath string
+	ReadOnly      bool
+	Type          string
```

Comment:
Was this supposed to be updated after https://github.com/microsoft/hcsshim/pull/1920 was merged or are we intentionally keeping it a string?

Reply:
I thought it would be good to keep this in sync with runtimespec mount struct which mount type as string and also keep it in sync with rest of hcsshim and containerd which uses string for types. Assigning or comparing values of mount types has been updated to use the const types to help understand/get a list of supported types after #1920